redisvl/extensions/llmcache/semantic.py

```diff
@@ -182,6 +194,14 @@ def delete(self) -> None:
         index."""
         self._index.delete(drop=True)
 
+    def drop(self, document_ids: Union[str, List[str]]) -> None:
```

Comment:
So for this, are we expecting users to pass the full redis key? or just the id portion (without prefix)? I think the terminology we try to use for this throughout the library is `id` when we are referring to the part without the prefix. So maybe we just use `ids` instead of `document_ids`?